app/controllers/api/cloud_networks_controller.rb

```diff
@@ -15,12 +15,13 @@ def options
 
       if params.key?("ems_id")
         ems = resource_search(params["ems_id"], :ext_management_systems, ExtManagementSystem)
-        raise BadRequestError, "No Cloud Network support for - #{ems.class}" unless defined?(ems.class::CloudNetwork)
+        klass = CloudNetwork.class_by_ems(ems)
 
-        klass = ems.class::CloudNetwork
-        raise BadRequestError, "No DDF specified for - #{klass}" unless klass.respond_to?(:params_for_create)
+        raise BadRequestError, "No Cloud Network support for - #{ems.class}" unless klass.supports?(:cloud_network)
```

Comment:
Sorry think I confused you, we don't have `cloud_networks` as a feature that we are tracking so you can't do `supports?(:cloud_network)` yet.

You can keep the `defined?(ems.class::CloudNetwork)` for now